### ISSUE TITLE
[PyTorch/BERT] Question about FusedLayerNorm

### ISSUE THREAD

**Issue by forresti** [NVIDIA/DeepLearningExamples#476]
### Related to **Model/Framework(s)**:
`DeepLearningExamples/PyTorch/BERT`

### Question

In [PyTorch/LanguageModeling/BERT/modeling.py](https://github.com/NVIDIA/DeepLearningExamples/blame/master/PyTorch/LanguageModeling/BERT/modeling.py), I see:

```
try:
    import apex
    #apex.amp.register_half_function(apex.normalization.fused_layer_norm, 'FusedLayerNorm')
    import apex.normalization
    from apex.normalization.fused_layer_norm import FusedLayerNormAffineFunction
    #apex.amp.register_float_function(apex.normalization.FusedLayerNorm, 'forward')
    #BertLayerNorm = apex.normalization.FusedLayerNorm
    APEX_IS_AVAILABLE = True
```

*My question is:* are `#apex.amp.register_half_function` and `#apex.amp.register_float_function` "[magic comments](https://stackoverflow.com/questions/6077479/how-does-the-magic-liness-in-python-work-when-specifying-encoding-in-python)", which have some affect on the code?

(Based on looking at the [apex](https://github.com/NVIDIA/apex) code, I think that these comments are simply commented-out code, but I wanted to double-check.) 

### tl;dr

If I remove the commented-out code as follows, is this code equivalent to the above code snippet?
```
try:
    import apex
    import apex.normalization
    from apex.normalization.fused_layer_norm import FusedLayerNormAffineFunction
    APEX_IS_AVAILABLE = True
```

### And finally...
If the lines I mentioned are old code that is commented out, could we delete them?

Also, this is tagged as bug, but perhaps it should actually be tagged as question. I don't seem to have control over the tagging.

**Comment by forresti** [comment]
(This is tagged as `bug`, but perhaps it should actually be tagged as `question`. I don't seem to have control over the tagging.)

**Comment by sharathts** [comment]
Nothing such. They are indeed commented out. Three will have to be put back when using AMP with opt level O1.

**Comment by forresti** [comment]
Thanks!
